### PR TITLE
Mopidy 3.4.2

### PR DESCRIPTION
**Pull request by monke0192**
Update Mopidy to 3.4.2. This also updates Python to 3.12, as I couldn't get Mopidy running under 3.11 for some reason.

### CODE REVIEW

File: Formula/mopidy-somafm.rb [hunk 1]
```diff
@@ -2,18 +2,18 @@ class MopidySomafm < Formula
   desc "Mopidy extension for playing music from SomaFM"
   homepage "https://github.com/AlexandrePTJ/mopidy-somafm"
   url "https://files.pythonhosted.org/packages/0e/6a/4cc6c5d1c813ec98343f24be7df3318acf118d2920ceeea6b6d31da1f1f0/Mopidy-SomaFM-2.0.2.tar.gz"
-  sha256 "0c2d1e9b192859f8c61e28760bc9c8341141c9fe76d577fedcab38251c4d3cb3"
+  sha256 "0c2d1e9b192859f8c61e28760bc9c83.1241c9fe76d577fedcab38251c4d3cb3"
```
Comment (jodal) [inline Formula/mopidy-somafm.rb]: The new dot in the hash here is probably not intentional.

File: Formula/mopidy.rb [hunk 2]
```diff
@@ -1,25 +1,21 @@
...
   head "https://github.com/mopidy/mopidy.git"
 
-  depends_on "python@3.11"
-  depends_on "gst-plugins-base"
-  depends_on "gst-plugins-good"
-  depends_on "gst-plugins-bad"
-  depends_on "gst-plugins-ugly"
-  depends_on "gst-python"
+  depends_on "python@3.12"
+  depends_on "gstreamer"
```
Comment (jodal) [inline Formula/mopidy.rb:9]: Replacing `gst-python` with `gstreamer` might be the right thing to do, but I haven't used Mac or Homebrew in a while, so I'm not familiar with any changes here.

Could you comment on the reasoning of this change?

File: Formula/mopidy.rb [hunk 3]
```diff
@@ -1,25 +1,21 @@
...
 
   resource "certifi" do
-    url "https://files.pythonhosted.org/packages/37/f7/2b1b0ec44fdc30a3d31dfebe52226be9ddc40cd6c0f34ffc8923ba423b69/certifi-2022.12.7.tar.gz"
-    sha256 "35824b4c3a97115964b408844d64aa14db1cc518f6562e8d7261699d1350a9e3"
+    url "https://files.pythonhosted.org/packages/71/da/e94e26401b62acd6d91df2b52954aceb7f561743aa5ccc32152886c76c96/certifi-2024.2.2.tar.gz"
```
Comment (jodal) [inline Formula/mopidy.rb:12]: Thanks for taking the time to update the deps too.

**Review by jodal — COMMENTED** [review]
Thanks! I'm not usually using Mac, so it is a long time between each time I sit down to test and update these, so any help here is much appreciated!

File: Formula/mopidy-beets.rb [hunk 4]
```diff
@@ -6,15 +6,15 @@ class MopidyBeets < Formula
   head "https://github.com/mopidy/mopidy-beets.git"
   revision 1
 
-  depends_on "python@3.11"
+  depends_on "python@3.12"
```
Comment (jodal) [inline Formula/mopidy-beets.rb:9]: Optimally, do the upgrade of Python in one commit and the upgrade of Mopidy in another.

File: Formula/mopidy.rb (hunk 2 above)
Comment (monke0192) [reply Formula/mopidy.rb:9]: All of the GStreamer plugins have been combined into one formula

File: Formula/mopidy-somafm.rb (hunk 1 above)
Comment (monke0192) [reply Formula/mopidy-somafm.rb]: Yeah, thanks for pointing that out

File: Formula/mopidy-beets.rb (hunk 4 above)
Comment (monke0192) [reply Formula/mopidy-beets.rb:9]: Got it 👍

**Comment by jodal** [comment]
One more thing I suddenly realized: I believe the formulas that got a new Python version but no version bump need to have revision set, like in https://github.com/mopidy/homebrew-mopidy/commit/a2049bd7103496548082a7af504af3acc6d99d1c. If not, people with existing installations will only get the Mopidy formula upgraded to 3.12, and not the extentions.

**Comment by monke0192** [comment]
> One more thing I suddenly realized: I believe the formulas that got a new Python version but no version bump need to have revision set, like in [a2049bd](https://github.com/mopidy/homebrew-mopidy/commit/a2049bd7103496548082a7af504af3acc6d99d1c). If not, people with existing installations will only get the Mopidy formula upgraded to 3.12, and not the extentions.

Yeah, just did that too

**Pull request merged**